arm-network/2017-03-01/swagger/applicationGateway.json

```diff
@@ -1282,10 +1308,28 @@
             "name": "ApplicationGatewayFirewallMode",
             "modelAsString": true
           }
+        },
+        "ruleSetType": {
+          "type": "string",
+          "description": "The type of the web application firewall rule set. Possible values are: 'OWASP'."
+        },
+        "ruleSetVersion": {
+          "type": "string",
+          "description": "The version of the rule set type."
+        },
+        "disabledRuleGroups": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ApplicationGatewayFirewallDisabledRuleGroup"
+          },
+          "description": "The disabled rule groups."
         }
       },
       "required": [
-        "enabled"
+        "enabled",
+        "firewallMode",
```

Comment:
The azure-sdk-for-net PR is currently still open since we have to coordinate multiple Network related PRs. I will changed the Version to a new major version as @DeepakRajendranMsft requested.